### DISCUSSION TITLE
[A-L] (22 maggio 2024) Esercizio 4

### DISCUSSION THREAD

**Discussion by Elia-Belli** [sapienzastudentsnetwork/progettazione-di-algoritmi#103] [Esercitazioni A-L]
![immagine](https://github.com/user-attachments/assets/9d5bdc6e-35cf-4e8d-a7d8-fd9bfc9b1354)

**Comment by daromacarlo** [comment]
![immagine](https://github.com/user-attachments/assets/5df8c9fa-d4a8-45cb-916c-49f8c1600798)
![immagine](https://github.com/user-attachments/assets/ef08ffe6-35b6-4429-9a5d-5f9890d2f8f7)
T(n) = O(n^2) dato che i due for annidati vengono eseguiti n volte e al loro interno ci sono operazioni di costo O(1).
